server.js

```diff
@@ -4,23 +4,20 @@ const express = require('express');
 const sa = require('superagent');
 const app = express();
 const PORT  = process.env.PORT || 3000;
+const requestProxy = require('express-request-proxy');
 
 app.use(express.static('./public'));
 console.log('my token: ' + process.env.GITHUB_TOKEN);
```

Comment:
What is this line doing here? site can't be opened with error "token undefined".